### PR TITLE
Fix #2311: Fix alignment of action buttons in cards on eval page

### LINKED ISSUE #2311: Fix staging travis build
**Description**:

The Travis build here(https://travis-ci.org/Cloud-CV/EvalAI/jobs/532140171#L5087) fails after the latest commit in master(https://github.com/Cloud-CV/EvalAI/commit/d9166ae8a20a1e4e35c851d99905223a91d7e0fb)

**Expected:**

Fix the Travis build

@sarthak212 Would you like to take this up since it fails after I merged your PR in master branch.

### PR DESCRIPTION
**Pull request by burnerlee**

### CODE REVIEW

**Comment by burnerlee** [comment]
Screenshots before the fix:
Screenshot from 2021-02-04 20-04-29
Screenshot from 2021-02-04 20-04-38

**Comment by burnerlee** [comment]
Screenshots after the fix:
Screenshot from 2021-02-04 20-01-59
Screenshot from 2021-02-04 20-02-11

**Review by Ram81 — CHANGES_REQUESTED** [review]
@burnerlee can we also change `upload` to `Upload` and `edit` to `Edit`

**Comment by Ram81** [comment]
@burnerlee can you also share screenshots of normal resolutions i.e 100% zoom

**Comment by burnerlee** [comment]
Screenshot from 2021-02-04 20-14-58
Screenshot from 2021-02-04 20-15-15

**Comment by burnerlee** [comment]
@Ram81 ^^

File: frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.scss [hunk 1]
```diff
@@ -6,6 +6,12 @@
   padding-right: 10px;
 }
 
+.ev-card-panel {
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.scss:9]: @burnerlee why are we adding new css class for `ev-card-panel`. Isn't this already present?

File: frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.scss [hunk 2]
```diff
@@ -29,4 +45,18 @@
   .tnc-pad {
     padding-left: 15px;
   }
+
+  .ev-card-panel {
```
Comment (Ram81) [inline frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.scss:49]: same here

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Comment by burnerlee** [comment]
Screenshot from 2021-02-04 20-39-40
Screenshot from 2021-02-04 20-39-58

**Review by Ram81 — APPROVED** [review]
LGTM

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**